app.js

```diff
@@ -375,3 +374,16 @@ function multipostModal() {
   });
 }
 /* eslint-enable no-unused-vars */
+
+// client side validation
+document.getElementById('form').addEventListener('submit', (e) => {
+  const emailValue = document.getElementById('email').value;
+  const emailError = document.getElementById('error-msg');
+  const regex = /^[a-z]/g;
+  if (regex.test(emailValue)) {
+    emailError.textContent = '';
+  } else {
+    e.preventDefault();
+    emailError.textContent = 'Email should be in lowercase';
+  }
+});
```

Comment:
- [ ] Everything works great, the only thing incomplete is the requirement that says that the error message should show up close to the "submit" button. Please make sure to display the error message close to  "submit" button